website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -6892,7 +6892,7 @@ Currently Python external transforms are limited to dependencies available in th
 #### 13.1.3. Creating cross-language Go transforms
 
 Go currently does not support creating cross-language transforms, only using cross-language
-transforms from other languages; see more at [Issue 21767](https://github.com/apache/beam/issues/21767).
+transforms from other languages; see more at [https://github.com/apache/beam/issues/21767](https://github.com/apache/beam/issues/21767).
```

Comment:
I think the existing one looks fine as-is, should we be changing it?

Reply:
Nope, fixed!